src/Illuminate/Notifications/Events/BroadcastNotificationCreated.php

```diff
@@ -130,6 +130,6 @@ public function broadcastAs()
     {
         return method_exists($this->notification, 'broadcastAs')
         ? $this->notification->broadcastAs()
-        : get_class($this->notification);
```

Comment:
Echo doesn't detect if passing the notification class? Plus this is a breaking change.